### PR TITLE
Mobile CSS - Menu Update

### PR DESCRIPTION
**Pull request by MiteyDrill**
- added script file
 - changes some html placements
 - AK page menu missing

### CODE REVIEW

**Comment by levimeahan** [comment]
Ah looks like this got the hotkey changes pulled in. I'm okay reviewing them together, but if you want to keep them separate in the future may need to create separate branches for each feature you work on

**Comment by MiteyDrill** [comment]
Okay, I didn't know the older PR would include changes after I submitted the request but it makes sense now. I'll try doing the branch change in the future. Might try the <div> thing later, gonna take a long break. Thanks Lsm:)!

**Comment by MiteyDrill** [comment]
I made some changes that should fix some issues I forgot to check for earlier

**Comment by MiteyDrill** [comment]
image
simple change fixed the buttons

**Comment by MiteyDrill** [comment]
@levimeahan I think I have a simple version of hotkeys set up here, it's hidden in the MobileCSS PR I can rename the PR if it helps

File: chat.php [hunk 1]
```diff
@@ -197,13 +197,13 @@ function chat() {
 			echo "
 				<tr>
 					<td style='text-align:center;'>
-					<div id='user_data_container' style='display: flex;flex-direction:row'>
+					<div id='user_data_container'>
```
Comment (levimeahan) [inline chat.php:200]: I think this is going to break it, why do we need to change this?

File: classes/system.php [hunk 2]
```diff
@@ -226,7 +226,13 @@ public function message($message, $force_message = false) {
...
     public function printMessage($force_display = false) {
         if(strlen($this->message) && (!$this->message_displayed || $force_display)) {
-            echo "<p class='systemMessage'>$this->message</p>";
+
+            $temp_msg = <<<HTML
+            <p class='systemMessage'> {$this->message} </p>
+            HTML;
+
+            echo $temp_msg;
+
```
Comment (levimeahan) [inline classes/system.php:235]:
```suggestion
echo "<p class='systemMessage'>{$this->message}</p>";
```

File: arena.php [hunk 3]
```diff
@@ -206,7 +206,18 @@ function arena() {
...
+
+				//can't seem to squeeze a <div> around $temp_time
+				//for some reason if there's a tag at the end
+				//it creates another empty <p> tag after the notification message
+				//confused at how and why this happens
+				$temp_msg = <<<HTML
+					<p>Please wait {$temp_time} more seconds!
+				HTML;
+
+				$system->message($temp_msg);
```
Comment (levimeahan) [inline arena.php:220]:
```suggestion
				$system->message("Please wait " . ($player->last_ai - (time() - $fight_timer)) . " more seconds!");
```
Let's sort this out in another PR

File: classes/system.php [hunk 4]
```diff
@@ -226,7 +226,7 @@ public function message($message, $force_message = false) {
     */
     public function printMessage($force_display = false) {
         if(strlen($this->message) && (!$this->message_displayed || $force_display)) {
-            echo "<p class='systemMessage'>$this->message</p>";
+echo "<p class='systemMessage'>{$this->message}</p>";
```
Comment (levimeahan) [inline classes/system.php:229]:
```suggestion
            echo "<p class='systemMessage'>{$this->message}</p>";
```

File: chat.php (hunk 1 above)
Comment (MiteyDrill) [reply chat.php:200]: Oh yeah sorry, I was testing this on the mobile view. Don't change this inline. mb

File: chat.php [hunk 5]
```diff
@@ -197,13 +197,13 @@ function chat() {
...
 					    <div style='flex-shrink:0;'>
 					        <img style='max-height:45px;max-width:45px;' src='{$userData['avatar_link']}' />
                         </div>
-						<div style='display:block;flex-grow:1;'>
-							<a href='{$system->links['members']}&user={$post['user_name']}' class='$class $statusType'>{$post['user_name']}</a><br />
-							<p style='margin: 1px 0 3px;'>
+						<div style='display:inline-block;flex-grow:1;'>
+							<a style='display:inline-block;' href='{$system->links['members']}&user={$post['user_name']}' class='$class $statusType'>{$post['user_name']}</a><br />
```
Comment (MiteyDrill) [inline chat.php:205]: I think this line will also affect chat

File: chat.php (hunk 1 above)
Comment (levimeahan) [reply chat.php:200]: Gotcha - You can feel free to migrate inline styles into the stylesheets so you can control per-layout, but need to add it to all the existing stylesheets if so. Should we just revert all these changes right here for now?

File: chat.php [hunk 6]
```diff
@@ -197,13 +197,13 @@ function chat() {
...
+							<p style='display:inline-block; margin: 1px 0 3px;'>
```
Comment (levimeahan) [inline chat.php:206]:
```suggestion
						<div style='display:block;flex-grow:1;'>
							<a style='display:inline-block;' href='{$system->links['members']}&user={$post['user_name']}' class='$class $statusType'>{$post['user_name']}</a><br />
							<p style='display:block; margin: 1px 0 3px;'>
```

File: chat.php (hunk 1 above)
Comment (levimeahan) [reply chat.php:200]:
```suggestion
					<div id='user_data_container' style='display: flex;flex-direction:row'>
```

File: chat.php [hunk 7]
```diff
@@ -197,13 +197,13 @@ function chat() {
...
-					<div id='user_data_container' style='display: flex;flex-direction:row'>
+					<div id='user_data_container'>
 					    <div style='flex-shrink:0;'>
 					        <img style='max-height:45px;max-width:45px;' src='{$userData['avatar_link']}' />
                         </div>
 						<div style='display:block;flex-grow:1;'>
-							<a href='{$system->links['members']}&user={$post['user_name']}' class='$class $statusType'>{$post['user_name']}</a><br />
-							<p style='margin: 1px 0 3px;'>
+							<a style='display:inline-block;' href='{$system->links['members']}&user={$post['user_name']}' class='$class $statusType'>{$post['user_name']}</a><br />
+							<p style='display:block; margin: 1px 0 3px;'>
```
Comment (levimeahan) [inline chat.php:206]:
```suggestion
							<p style='margin: 1px 0 3px;'>
```

**Review by levimeahan — APPROVED** [review]

**Pull request merged**